src/applications/edu-benefits/0994/containers/IntroductionPage.jsx

```diff
@@ -143,4 +143,4 @@ export class IntroductionPage extends React.Component {
   }
 }
 
-export default IntroductionPage;
+export default IntroductionPage;
```

Comment:
If you undo this line change it should fix the build failing due to lint issues

Reply:
Great, will update that